hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieCLIUtils.scala

```diff
@@ -40,8 +41,9 @@ object HoodieCLIUtils extends ProvidesHoodieConfig{
       .setConf(sparkSession.sessionState.newHadoopConf()).build()
     val schemaUtil = new TableSchemaResolver(metaClient)
     val schemaStr = schemaUtil.getTableAvroSchema(false).toString
+    val table = StringUtils.join(metaClient.getTableConfig.getDatabaseName, ".", metaClient.getTableConfig.getTableName)
     val finalParameters = HoodieWriterUtils.parametersWithWriteDefaults(
-      buildHoodieConfig(getHoodieCatalogTable(sparkSession, metaClient.getTableConfig.getTableName)) ++ conf)
```

Comment:
Be caution that `metaClient.getTableConfig.getDatabaseName` does not have default value.

Reply:
Judgment has been added

Reply:
Can the `databaseName` be null?

Reply:
It will not be null. Because when creating a hudi table, if there is no database, the current default database will be used as the databaseName. So `metaClient.getTableConfig.getDatabaseName` usually returns `default`.

We can see the part from here:
https://github.com/apache/hudi/blob/master/hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/catalyst/catalog/HoodieCatalogTable.scala#L187

Reply:
So the database name can never be empty right?

Reply:
Yes, let me remove the code about empty judgment

Reply:
updated